From 09335efcbfb04da83090f9935b3eda1def31e511 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Sun, 23 Jun 2024 12:32:23 -0700
Subject: [PATCH] Docs/mm/allocation-profiling: mark 'Theory of operation' as
 chapter

'Theory of operation' part of allocation-profiling document is
apparently a chapter.  However, it is mistakenly marked as a document
title.  As a result, rendered mm document index page shows two items for
the document.  Fix it to be marked as a chapter.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/allocation-profiling.rst | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Documentation/mm/allocation-profiling.rst b/Documentation/mm/allocation-profiling.rst
index d3b733b41ae6..ffd6655b7be2 100644
--- a/Documentation/mm/allocation-profiling.rst
+++ b/Documentation/mm/allocation-profiling.rst
@@ -46,7 +46,6 @@ Example output::
          55M     4887 mm/slub.c:2259 func:alloc_slab_page
         122M    31168 mm/page_ext.c:270 func:alloc_page_ext
 
-===================
 Theory of operation
 ===================
 
-- 
2.39.2

